<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfigurationPage</class>
 <widget class="QWidget" name="GeneralConfigurationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Général</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Apparence</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="m_use_system_color_cb">
        <property name="text">
         <string>Utiliser les couleurs du système</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Collections d'éléments</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Chemin de la collection utilisateur</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Chemin de la collection commune</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m_common_elmt_path_cb">
        <item>
         <property name="text">
          <string>Par defaut</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parcourir...</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m_custom_elmt_path_cb">
        <item>
         <property name="text">
          <string>Par defaut</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parcourir...</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>(Recharger les collections d'éléments pour appliquer les changements)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Projets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="2">
       <widget class="QPushButton" name="m_font_pb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sauvegarde automatique des projets (appliqué au prochain lancement de QElectroTech)</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="m_folio_list_pb">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="m_save_label_paste">
        <property name="text">
         <string>Ne pas conserver les labels des éléments lors des copier coller</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_use_tab_mode_rb">
        <property name="text">
         <string>Utiliser des onglets (appliqué au proc&amp;hain lancement de QElectroTech)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="m_use_gesture_trackpad">
        <property name="text">
         <string>Utiliser les gestes du pavé tactile</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Police des champs textes des pages sommaires</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Police des champs de texte</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="m_export_terminal">
        <property name="text">
         <string>Exporter les bornes dans la nomenclature</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="m_autosave_sb">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="specialValueText">
         <string>Désactivé</string>
        </property>
        <property name="suffix">
         <string comment="minute"> min</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="m_use_windows_mode_rb">
        <property name="text">
         <string>Utiliser des fenêtres (appliqué au prochain lancement de &amp;QElectroTech)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="m_zoom_out_beyond_folio">
        <property name="text">
         <string>Autoriser le dézoom au delà du folio</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="m_use_folio_label">
        <property name="text">
         <string>Utiliser les labels de folio à la place de leurs ID</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Gestion des éléments</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="m_highlight_integrated_elements">
        <property name="text">
         <string>Mettre en valeur dans le panel les éléments fraîchement intégrés</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Chaque élément embarque des informations sur ses auteurs, sa licence, ou tout autre renseignement que vous jugerez utile dans un champ libre.
Vous pouvez spécifier ici la valeur par défaut de ce champ pour les éléments que vous créerez :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_default_elements_info"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Langue</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="m_lang_cb"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Appliqué au prochain lancement de QElectroTech</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
